From 30df3ab3c92d0e7854c54df6ae66f4aa6eb5b3d2 Mon Sep 17 00:00:00 2001
From: Heiko Carstens <hca@linux.ibm.com>
Date: Thu, 6 Aug 2026 15:00:50 +0200
Subject: [PATCH 11352/15333] s390/block: Enable CONTEXT_ANALYSIS

All drivers in drivers/s390/block pass clang's compile time context
analysis. Therefore enable CONTEXT_ANALYSIS.

Signed-off-by: Heiko Carstens <hca@linux.ibm.com>
Acked-by: Stefan Haberland <sth@linux.ibm.com>
Link: https://patch.msgid.link/20260806130050.2057443-3-hca@linux.ibm.com
Signed-off-by: Jens Axboe <axboe@kernel.dk>
---
 drivers/s390/block/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/s390/block/Makefile b/drivers/s390/block/Makefile
index a0a54d2f063f..3dd1a3ce7ed1 100644
--- a/drivers/s390/block/Makefile
+++ b/drivers/s390/block/Makefile
@@ -3,6 +3,8 @@
 # S/390 block devices
 #
 
+CONTEXT_ANALYSIS := y
+
 dasd_eckd_mod-objs := dasd_eckd.o dasd_3990_erp.o dasd_alias.o
 dasd_fba_mod-objs  := dasd_fba.o
 dasd_diag_mod-objs := dasd_diag.o
-- 
2.55.0

